src/Simulation.h

```diff
@@ -12,11 +12,17 @@ along with this program, also can be found at <http://www.gnu.org/licenses/>.
 #include "System.h"
 #include "StaticVals.h"
 #include "BasicTypes.h"
-
+#include "GOMC_Config.h"    //For PT
+#if GOMC_LIB_MPI
+#include "ParallelTemperingPreprocessor.h"
+#endif
 class Simulation
 {
 public:
   explicit Simulation(char const*const configFileName);
+  #if GOMC_LIB_MPI
+  explicit Simulation(char const*const configFileName, MultiSim *& multisim);
+  #endif
```

Comment:
@go2432 Can you come up with another alternative solution?

Reply:
explicit Simulation(char const*const configFileName, MultiSim const*const& multisim = NULL);